Sources/ParseSwift/Coding/ParseEncoder.swift

```diff
@@ -0,0 +1,259 @@
+//
+//  ParseEncoder.swift
+//  ParseSwift
+//
+//  Created by Pranjal Satija on 7/20/20.
+//  Copyright © 2020 Parse. All rights reserved.
+//
+
+// This rule doesn't allow types with underscores in their names.
+// swiftlint:disable type_name
+
+import Foundation
+
+// MARK: ParseEncoder
+public struct ParseEncoder {
```

Comment:
I think this should be marked `internal`, even when an external framework conforms to the storage protocol they should ask ParseSwift to encode/decode. This gives us the flexibility of modifying the ParseEncoder as we see fit as leaving it public enables developers to think they can override and make their own versions of the ParseEncoder

Reply:
Sounds good, I'll make it `internal`.